jenkins/gce-custom-images/prepare_gce_snapshot.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+#
+# This script sets up a custom GCE snapshot for VMs provisioned by Leonardo.
+# The service account used to run the parent script must have the following permissions:
+# TODO: Verify the list below is correct when the snapshot creation is Jenkinsified
+# [  "roles/compute.admin",
+#    "roles/iam.serviceAccountUser",
+#    "roles/storage.objectViewer",
+
+#
+# Constants and Global Vars
+#
+
+# The versions below don't matter; they are replaced by the Jenkins job
+terra_jupyter_base="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:0.0.19"
```

Comment:
would be nice to switch these to some function that reads from automation tests's `reference.conf`...but doesn't have to be in this PR

Reply:
I think how it works is the Jenkins job replaces the version strings with `sed` when it builds the image

Reply:
👍🏼   will add a TODO